packages/core/src/__integrationtests__/Ctypes.spec.ts

```diff
@@ -77,12 +71,11 @@ describe('When there is an CtypeCreator and a verifier', () => {
     )
     await submitTx(authorizedStoreTx, paymentAccount)
 
-    expect(CType.fromChain(await api.query.ctype.ctypes(ctype.hash))).toBe(
-      ctypeCreator.uri
-    )
-    await expect(CType.verifyStored(ctype)).resolves.not.toThrow()
-
-    ctype.owner = ctypeCreator.uri
+    expect(
+      CType.fromChain(
+        await api.query.ctype.ctypes(CType.getCTypeHashFromId(ctype.$id))
```

Comment:
Even if it would be just an alias, I think it would make for a consistent code to use `CType.toChain()` here

Reply:
we do have `CType.toChain` however for serialising the CType for submission. So we'd have to call id `CType.idToChain`

Reply:
Sounds good!